### PR TITLE
update node-soap

### LINKED ISSUE #20: Error undefined:undefined (update soap)
I've previously gotten an error with the status code `undefined:undefined`. This is a problem inside `node-soap` and they've addressed it in [this commit](https://github.com/vpulim/node-soap/commit/605ec59130d020417d83ff0c6a8006951042acc1), which we should get by updating to v0.11.0.

I'm not sure if updating will have any breaking changes, but I can research, try it out, and then submit a PR.

### PR DESCRIPTION
**Pull request by JuanCaicedo**
Updates `node-soap` which fixes #20!

I found a breaking change in one service/method:

Service | Method 
--------|--------
LineItemService | createLineItems

This seemed to be because my test line item had a property
```json
  "targeting": {
    "geoTargeting": {
      "targetedLocations": [null]
    },
```
Which was fixed by changing `targetedLocations` to `[]`

I tested the following services/methods and found no problems:

Service | Method 
--------|--------
LineItemService | getLineItemsByStatement
CustomTargetingService | getCustomTargetingKeysByStatement
CustomTargetingService | getCustomTargetingValuesByStatement
InventoryService | getAdUnitsByStatement
OrderService | getOrdersByStatement
CompanyService | getCompaniesByStatement
LabelService | getLabelsByStatement
CreativeService | getCreativesByStatement
CreativeService | createCreatives
OrderService | createOrders
LineItemCreativeAssociationService | createLineItemCreativeAssociations
LineItemCreativeAssociationService | updateLineItemCreativeAssociations
LineItemCreativeAssociationService | getLineItemCreativeAssociationsByStatement
ReportService | runReportJob
ReportService | getReportJobStatus
ReportService | getReportDownloadURL

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,6 +33,9 @@
   },
   "homepage": "https://github.com/ShinyAds/node-google-dfp",
   "keywords": [
-    "Google", "DFP", "DoubleClick", "adserver"
+    "Google",
+    "DFP",
+    "DoubleClick",
+    "adserver"
```
Comment (JuanCaicedo) [inline package.json]: This reformatting was done by npm.

**Comment by ajsouza** [comment]
Thanks

**Comment by JuanCaicedo** [comment]
@ajsouza Thanks for accepting this! Would it be possible for you to release a new version with this?

**Comment by ajsouza** [comment]
Done

**Comment by JuanCaicedo** [comment]
Great, thank you!

**Pull request merged**